gitpod-io/openvscode-server
# Add a Get Started with Gitpod walkthrough

**How to test**
1. Preview in web
1. On the Welcome screen, look for the new walkthrough on the right side
1. Review the walkthrough, click links and verify everything works as expected

To reset the walkthrough progress, use the command palette and search for "Developer: Reset Welcome Page Walkthrough Progress"
